src/markdown-pages/install-with-kurl/reducing-storage.md

```diff
@@ -0,0 +1,211 @@
+The procedure to do so is long and painful, but it does exist.
+
+### Identifying OSDs
+The first step is to determine what OSD number corresponds to the block device you wish to be freed up.
+
+With a shell into the tools deployment from `kubectl exec -it -n rook-ceph deployment/rook-ceph-tools -- bash`, we can get a list of disks that ceph is using, and which disk is on which host.
+
+```
+[rook@rook-ceph-tools-6fb84b545-js4rg /]$ ceph osd tree
+ID  CLASS  WEIGHT   TYPE NAME                     STATUS  REWEIGHT  PRI-AFF
+-1         0.58597  root default
+-3         0.39067      host laverya-rook-main
+ 0    ssd  0.09769          osd.0                     up   1.00000  1.00000
+ 1    ssd  0.09769          osd.1                     up   1.00000  1.00000
+ 2    ssd  0.19530          osd.2                     up   1.00000  1.00000
+-5         0.19530      host laverya-rook-worker
+ 3    ssd  0.04880          osd.3                     up   1.00000  1.00000
+ 4    ssd  0.14650          osd.4                     up   1.00000  1.00000
+[rook@rook-ceph-tools-6fb84b545-js4rg /]$ ceph osd df
+ID  CLASS  WEIGHT   REWEIGHT  SIZE     RAW USE  DATA     OMAP  META     AVAIL    %USE   VAR   PGS  STATUS
+ 0    ssd  0.09769   1.00000  100 GiB   13 GiB   13 GiB   0 B   52 MiB   87 GiB  13.45  0.76   37      up
+ 1    ssd  0.09769   1.00000  100 GiB   25 GiB   25 GiB   0 B   93 MiB   75 GiB  25.38  1.44   47      up
+ 2    ssd  0.19530   1.00000  200 GiB   32 GiB   32 GiB   0 B  124 MiB  168 GiB  16.17  0.92   85      up
+ 3    ssd  0.04880   1.00000   50 GiB  9.4 GiB  9.4 GiB   0 B   39 MiB   41 GiB  18.88  1.07   45      up
+ 4    ssd  0.14650   1.00000  150 GiB   25 GiB   25 GiB   0 B  100 MiB  125 GiB  16.75  0.95  115      up
+                       TOTAL  600 GiB  106 GiB  105 GiB   0 B  407 MiB  494 GiB  17.62
+MIN/MAX VAR: 0.76/1.44  STDDEV: 4.05
+```
+If this is enough to identify your disk already, fantastic!
+For instance, a 200GB disk on `laverya-rook-main` would be osd.2 above.
+
+If this is not enough - for instance if you have multiple disks of the same size on the same instance - you can use kubectl from the host.
+You can get the list of rook-ceph OSDs with `kubectl get pods -n rook-ceph -l ceph_daemon_type=osd`, and then check each one by searching the describe output for `ROOK_BLOCK_PATH`.
+
+```
+kubectl describe pod -n rook-ceph rook-ceph-osd-0-6cf7c5cb7-z7c8p | grep ROOK_BLOCK_PATH
+      DEVICE="$ROOK_BLOCK_PATH"
+      ROOK_BLOCK_PATH:              /dev/sdb
+      ROOK_BLOCK_PATH:              /dev/sdb
+```
+From this we can tell that the OSD using /dev/sdb is OSD 0.
+
+### Draining OSDs
+Once this is known, there are many commands to be run within the rook-ceph-tools deployment.
+You can get a shell to this deployment with `kubectl exec -it -n rook-ceph deployment/rook-ceph-tools -- bash`.
+
+Once you have determined the OSD to be removed, you can tell ceph to relocate data off of this disk with `ceph osd reweight 1 0`.
```

Comment:
Could be a little jarring switching from osd.2 above to osd.1 here at the very least clarifying that you're trying to remove osd.1 in this line would be helpful if someone just did the 'easy' identification above and then jumped to here.

Reply:
where do I mention osd.2?

I can change that too

Reply:
osd.2 is the example above for the "simple" way of finding the disk. I agree if it were just consistently osd.1 the whole time that would also address this.